dobi.yaml

```diff
@@ -0,0 +1,93 @@
+
+meta:
```

Comment:
> But more to the point: what I hear is confirmation that dobi does not abstract installation of packages on the base platform, but helps to install packages within the container.

Yes, `dobi` is a tool for working with docker containers.

> If working on a redhat-based container is a goal,

It's not. The container is always `ubuntu` (or `alpine`). But it can run on any host (not just ubuntu).

> Doing this at the fabfile level becomes independently useful.

Sure, you could attempt to support and maintain the install on every host... or you could do it just once in a container! Why do it more than once?

> If we decide we can stick to an ubuntu container, then dobi abstracting package installation within the container is irrelevant, no? 

No, because you can run that ubuntu container on any host (windows, osx, ubuntu, arch, redhat, etc).

It means that *everyone* gets exactly the same thing, instead of having to maintain multiple copies of things, and then trying to figure out why things happen to not work for platform x when they work on y.